examples/workspace/WORKSPACE

```diff
@@ -0,0 +1,38 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "rules_cc",
```

Comment:
I think it's too early to load/use `rules_cc` for rules yet, especially with WORKSPACE without MVS.

Reply:
I will remove the explicit declaration here.

Note that folks are already loading these through the toolchain's deps.bzl though.